mne/viz/montage.py

```diff
@@ -1,20 +1,22 @@
 """Functions to plot EEG sensor montages or digitizer montages."""
-import numpy as np
+from ..utils import check_version
+from . import plot_sensors
 
-from .utils import plt_show
 
-
-def plot_montage(montage, scale_factor=20, show_names=False, show=True):
+def plot_montage(montage, kind='topomap', scale_factor=20, show_names=True,
+                 show=True):
```

Comment:
see @agramfort's unaddressed comment above about new parameter insertion at the end (lest we change API)